wormhole-connect/src/views/Redeem/SendTo.tsx

```diff
@@ -126,12 +126,11 @@ function SendTo() {
       ? inProgress && !transferComplete
       : !transferComplete;
   const manualClaimText =
-    transferComplete || txData.payloadID === Route.RELAY
+    transferComplete || txData.payloadID === Route.RELAY // todo: should be the other enum, should be named better than payload id
```

Comment:
Per earlier comment.  Yes, payload ID is different.  Can make a `txData.isAutomatic` value